packages/commonwealth/client/scripts/views/modals/AuthModal/types.ts

```diff
@@ -20,7 +20,9 @@ export type ModalVariantProps = {
     | ChainBase.Ethereum
     | ChainBase.CosmosSDK
     | ChainBase.Solana
-    | ChainBase.Substrate;
+    | ChainBase.Substrate
+    | ChainBase.Skale;
```

Comment:
we dont need to add `ChainBase.Skale;` here

Reply:
without it CI/CD are  failing . 
` client/scripts/views/pages/CreateCommunity/steps/CommunityTypeStep/CommunityTypeStep.tsx(193,9): error TS2322: Type 'ChainBase.CosmosSDK | Cha…
│   Type 'ChainBase.Skale' is not assignable to type 'ChainBase.CosmosSDK | ChainBase.Substrate | ChainBase.Ethereum | ChainBase.Solana | undefi…
└─ Failed in 23s at /Users/Malikkaleem/Documents/company_project/commonwealth/packages/commonwealth`

Reply:
Malike is correct, this is not a valid change. If CI/CD is failing then there is a bug elsewhere because Skale should be treated as `ChainBase.Ethereum`.